techniques/html/H48.html

```diff
@@ -118,8 +122,8 @@ <h3>Example 1: A list showing steps in a sequence</h3>
 &lt;/ol&gt;</code></pre>
       </section>
       <section class="example" id="example-2-a-grocery-list-">
-         <h3>Example 2: A grocery list </h3>
-         <p>This example shows an unordered list of items to buy at the store.</p>
+         <h3>事例 2: 材料のリスト</h3>
```

Comment:
既存訳ですが、原文が grocery なので、「材料」というよりは「食料品」ですかね。お店で買うものリストなので、「食材」でもよいかもしれません。
```suggestion
         <h3>事例 2: 食材のリスト</h3>
```